static/app/views/detectors/components/detectorListRow.tsx

```diff
@@ -65,14 +66,17 @@ export function DetectorListRow({
         </CellWrapper>
         <StyledGraphCell />
       </Flex>
+      <CellWrapper className="type">
```

Comment:
just curious - are we using these as css classes or just data info?

Reply:
these classes are currently being used to responsively hide certain columns, but we'll definitely need a better approach.